docs/getting-started/release-notes/v2_3_0.md

```diff
@@ -12,36 +12,60 @@ Zowe Version 2.3.0 contains the enhancements that are described in the following
 
 ### Zowe installation and packaging
 
-
 ### Zowe Application Framework
 
-
 ### Zowe API Mediation Layer
 
-
 ### Zowe CLI
 
 #### Zowe CLI (Core)
 
-### Zowe Explorer
+- Added the browser-view option to the `zowe zos-files compare data-set` command to compare two data sets, and display the differences in the browser. ([#1443](https://github.com/zowe/zowe-cli/issues/1443))
+- Added the command `zowe zos-files compare local-file-data-set` to compare a local file and a data set, and display the differences in the browser and terminal. ([#1444](https://github.com/zowe/zowe-cli/issues/1444))
+- Added the command `zowe zos-files compare uss-files` to compare two uss files, and display the differences in the browser and terminal. ([#1445](https://github.com/zowe/zowe-cli/issues/1445))
+- Added the command `zowe zos-files compare local-file-uss-file` to compare a local file and a uss file, and display the differences in the browser and terminal. ([#1446](https://github.com/zowe/zowe-cli/issues/1446))
+- Added the command `zowe zos-files compare spool-dd` to compare two spool dds, and display the differences in the browser and terminal. ([#1447](https://github.com/zowe/zowe-cli/issues/1447))
+- Added the command `zowe zos-files compare local-file-spool-dd` to compare a local file and a spool dd, and display the differences in the browser and terminal. ([#1448](https://github.com/zowe/zowe-cli/issues/1448))
+- Added the `ZOWE_CLI_PLUGINS_DIR` environment variable to override the location where plug-ins are installed. ([#1483](https://github.com/zowe/zowe-cli/issues/1483))
+- Added the `zowe zos-files compare data-set` command to compare two data sets, and display the differences in the terminal. ([#1442](https://github.com/zowe/zowe-cli/issues/1442))
+
+#### Zowe CLI Imperative Framework
+
+- Added `ZOWE_CLI_PLUGINS_DIR` environment variable to override the location where plug-ins are installed.  ([Zowe CLI #1483](https://github.com/zowe/zowe-cli/issues/1483))
 
+- Added Diff utility features for getting differences between two files and open diffs in browser. Also added web diff generator for creating web diff dir at the CLI home.
 
 ## Bug fixes
 
 Zowe Version 2.3.0 contains the bug fixes that are described in the following topics.
 
 ### Zowe API Mediation Layer
 
-
 ### Zowe CLI
 
-Zowe CLI and related components contain the following bug fixes.
+Zowe CLI and related components contain the following bug fixes:
+
+- Updated Imperative to include bug fixes in version 5.5.1.
+- Updated Imperative to include bug fixes in version 5.5.0.
+- Renamed `download data-set-matching` to `download data-sets-matching`. The old name still exists as an alias.
+- Fixed output of `download data-sets-matching` being printed twice when some data sets fail to download.
+- Updated Imperative to fix error when installing plug-ins that do not define profiles.
+- Altered the `zowe daemon disable` command to kill only the daemon running for the current user.
```

Comment:
I think the 2nd and 3rd lines beginning with "Updated Imperative" can be removed, since they are covered by the 1st one:
* ✔️ Updated Imperative to include bug fixes in version 5.5.1.
* ❌ Updated Imperative to include bug fixes in version 5.5.0.
* ❌ Updated Imperative to fix error when installing plug-ins that do not define profiles.

Reply:
updated